app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetHealthFragment.java

```diff
@@ -35,7 +35,7 @@
     private FragmentInfoPetHealthBinding binding;
     private TextView statisticTitle;
     private MaterialButton btnAddNewStatistic;
-    private BarChart barChart;
+    private static BarChart barChart;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5541824)